.stylelintrc.json

```diff
@@ -0,0 +1,102 @@
+{
+  "extends": "stylelint-config-recommended",
+  "plugins": [
+    "stylelint-order"
+  ],
+  "ignoreFiles": [
```

Comment:
We'll need to exclude the prismjs styles indefinitely.

I ignored the list components for the moment because @m6jones and others are in the midst of lots of changes under tight timeline and the linting fixes will be disruptive for their PR.